CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+CHANGELOG
+=========
+
+0.7.0
```

Comment:
I would prefer to keep this in a section `UNRELEASED` and change the number on release.

Reply:
Fixed.